.github/workflows/deploy.yml

```
@@ -0,0 +1,33 @@
+# yamllint --format github .github/workflows/deploy.yml
+---
+name: deploy
+
+# We deploy on a release versions, regardless of if the commit is documentation-only or not.
+#
+# See https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
+on:
+  push:
+    tags: '[0-9]+.[0-9]+.[0-9]+**'  # Ex. 1.2.3
+    branches: master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04  # newest available distribution, aka focal
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1  # full history is not needed
+      - name: Set up Ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.6.x
```

```suggestion
          ruby-version: 2.7.x
```

Seems like it repros what we have in https://github.com/openzipkin/zipkin-ruby/blob/master/.travis.yml#L28